modules/importing-repositories/procedure-data-handoff-and-custom-workflow-design.adoc

```diff
@@ -0,0 +1,17 @@
+:_mod-docs-content-type: PROCEDURE
+
+[id="full-data-handoff-and-custom-workflow-design"]
+
+= Data handoff and custom workflow design
+
+When you configure the Bulk Import plugin by setting the importAPI field to `scaffolder`, the Bulk Import Backend ensures all necessary context is passed directly to the Scaffolder API.
+
+Your control over input data:: The Orchestrator workflow receives the input data required to automate processing for every selected repository. For each repository in that list, the bulk import launches the scaffolder template once and passes the data for that single repository to the template run.
+
+As an administrator, you can define the Scaffolder template workflow and structure the workflow to do the following:
+
+Consume the input:: The Scaffolder template receives the repository data as template parameters for the current execution.
+
+Iterate and process:: The orchestrator handles iterating through the repository list and launches the template for each repository. Within a single template run, implement the custom logic for that single repository. For example, create the `catalog-info.yaml` if needed, run compliance checks, then register with the catalog.
```

Comment:
the other sections describe the actions from the Scaffolder's point of view. can you align the wording across all sections, please? from a JTBD standpoint, describing the options from the user's point of view would be better.